update-app-doc.xsl

```diff
@@ -285,9 +285,20 @@
   </xsl:template>
 
   <!-- toplevel boilerplate -->
-  <xsl:template match="zabbix_export"># Zabbix <xsl:value-of select="$appName"/> monitoring<xsl:text>
+  <xsl:template match="zabbix_export">
```

Comment:
Did you commit the results of this change somewhere or am I missing something? Didn't we agree on getting rid of these underscores at the source?

Reply:
> Did you commit the results of this change somewhere or am I missing something?
- https://github.com/radiorabe/rabe-zabbix/pull/29/commits/edf9f4310121aa5c0f68c1126e5525f17b13273e#diff-a9e5e544b49e4b25ef08dbab2ea4916dR1
- https://github.com/radiorabe/rabe-zabbix/pull/29/commits/edf9f4310121aa5c0f68c1126e5525f17b13273e#diff-f66954e77df16b3208191299ba9a93eeR1


> Didn't we agree on getting rid of these underscores at the source?

Yes, we did, but forgot to re-address them within the `Makefile`. The `Makefile` takes the names from the directories which have and underscore in their names ([`appDir="app/${appName// /_}"`](https://github.com/radiorabe/rabe-zabbix/blob/master/README.md#adding-an-app-template), that's why we need to replace them again.